examples/durable/LongRunningHttpApp/host.json

```diff
@@ -0,0 +1,7 @@
+{
+  "version": "2.0",
+  "managedDependency": {
+    "enabled": true
+  },
+  "functionTimeout": "02:00:00"
```

Comment:
Is the function timeout 2 hours?

Reply:
The default `functionTimeout` value is 30 minutes. For the "Long-Running HTTP Request" scenario, we need to be able to run a function for 1 hour. However, the default `functionTimeout` prevents this from happening, so I'm setting it to 2 hours just as a sample demonstrating how this can be done. The actual time that this specific function takes depends on the input data (see `$InputData.DurationSeconds` passed to LongRunningActivity).

I realize these samples would benefit from some additional explanations. However, this is not a user-facing documentation yet, and many things will change before we get to that point, so I would do this a bit later.

Reply:
Sounds good.

Reply:
Sounds good.